redfish-core/lib/log_services.hpp

```diff
@@ -6728,9 +6728,21 @@ static LogParseError
     // Fill the MessageArgs into the Message
     if (messageArgs.size() > 0)
     {
+        if (messageArgs[0] != "USYS_CONFIG")
+        {
+            BMCWEB_LOG_WARNING << "Unexpected audit log entry type: "
+                               << messageArgs[0];
+        }
+
         int i = 0;
         for (auto messageArg : messageArgs)
         {
+            if (messageArg == nullptr)
```

Comment:
If this happens, how should the original `messageArgs` be used at L6764?

```
    logEntryJson["MessageArgs"] = std::move(messageArgs);
```

Should `messageArgs` be changed with "" (after reconstructing it)?

Reply:
Oh, good point. Yes it should probably also be adjusted to use a null string so the GUI doesn't have to worry about getting a null entry.